bloom-housing/bloom
# 2098/fix partner application grid

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2098)


## Description

This started as a rehash of https://github.com/bloom-housing/bloom/pull/2188 but somethings came up. Mainly there are big limitations to the nestjs-typeorm-pagination library that was causing issues where it was being used

I resolved this for partner's applications grid for a listing, and partner's user grid  

## How Can This Be Tested/Reviewed?
Please see the previous PR for instructions and comments, but a quick run is:

go to a listing on the partner's side and view the applications associated with that listing. The grid should display correct with records and pagination working properly

You should also test on the partner's side the user grid should now display data correctly which can be compared to what is displayed on the dev currently 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
